### PR TITLE
Update meteor version support

### PR DESCRIPTION
**Pull request by curzolapierre**
- meteor v2.0 is now supported

Fix [OPSB-215]

[OPSB-215]: https://scalingo.atlassian.net/browse/OPSB-215

### CODE REVIEW

**Review by Soulou — CHANGES_REQUESTED** [review]
I don't see where you modify the code to run `main.js` Have you forgotten some code?

**Comment by curzolapierre** [comment]
It was already present below the modified condition, here: https://github.com/Scalingo/nodejs-buildpack/blob/77f725217e27c97f44538bce5656470955884935/bin/meteor.sh#L131-L133

I thought to add a new one, something like:
```bash
  if [ "$major" -ge 2 ]; then
    cat << EOF > "${build_dir}/.start-meteor-app"
cd .app-build/bundle
exec node \$NODE_BOOT_FLAGS main.js
EOF
  fi
```
So, it is a duplicate of code but in the future versions we will probably do something like this.
Do I add it already for this PR ?

File: bin/meteor.sh [hunk 1]
```diff
@@ -96,14 +120,20 @@ EOF
 create_meteor_startup_file() {
```
Comment (Soulou) [inline bin/meteor.sh:120]: The condition is a bit weird

Why not

If major == 1 → boot.js program.json
else  main.js

and that's it?

**Review by Soulou — CHANGES_REQUESTED** [review]

File: bin/meteor.sh (hunk 1 above)
Comment (curzolapierre) [reply bin/meteor.sh:120]: like what was done on this commit https://github.com/Scalingo/nodejs-buildpack/pull/42/commits/77f725217e27c97f44538bce5656470955884935 ?

Comment (Soulou) [reply bin/meteor.sh:120]: OOOH you're right I didn't get that the `main.js` was already mentionned (it was hidden from the diff)

**Review by Soulou — APPROVED** [review]
LGTM

**Pull request merged**